document/core/exec/instructions.rst

```diff
@@ -182,6 +182,494 @@ Where the underlying operators are non-deterministic, because they may return on
    \end{array}
 
 
+.. index:: SIMD instruction
+   pair: execution; instruction
+   single: abstract syntax; instruction
+.. _exec-instr-simd:
+
+SIMD Instructions
+~~~~~~~~~~~~~~~~~~~~
+
+SIMD instructions are defined in terms of generic numeric operators applied lane-wise based on the :ref:`shape <syntax-simd-shape>`.
+
+.. math::
+   \begin{array}{lll@{\qquad}l}
+   \X{op}_{t\K{x}N}(n_1,\dots,n_k) &=&
+   \simdof_{t\K{x}N}(
+   op_t(
+   \simdto_{t\K{x}N}(n_1))
+   ~\dots~
+   \simdto_{t\K{x}N}(n_k)
+   )
+   \end{array}
+
+.. note::
+   For example, the result of instruction :math:`\K{i32x4}.\ADD` applied to operands :math:`i_1, i_2`
+   invokes :math:`\ADD_{\K{i32x4}}(i_1, i_2)`, which maps to
+   :math:`\simdof_{\K{i32x4}}(\ADD_{\I32}(i_1^+, i_2^+))`,
+   where :math:`i_1^+` and :math:`i_2^+` are sequences resulting from invoking
+   :math:`\simdto_{\K{i32x4}}(i_1)` and :math:`\simdto_{\K{i32x4}}(i_2)`
+   respectively.
+
+
+..
+  .. math::
+     \begin{array}{lll@{\qquad}l}
```

Comment:
6. This commented section is how I defined simd operations *if* we did not lift numeric operations to work on input sequences. I think it is harder to read, many \dots, and more ambiguous.